.github/workflows/deploy.yml

```diff
@@ -24,7 +23,7 @@ jobs:
         uses: changesets/action@v1
         with:
           title: Version Packages (${{ github.ref_name }})
-          publish: yarn run deploy --tag ${{ github.ref_name }}
+          publish: yarn run deploy --tag ${{ github.ref_name == vars.LATEST_STABLE_VERSION && 'latest' || github.ref_name }}
```

Comment:
We need 2 tags for the latest `--tag ${{ github.ref_name}},latest`

Reply:
It does support it as a comma separated list no? This should work `yarn run deploy --tag ${{ github.ref_name}},latest`

Reply:
Unfortunately, no. This gets treated as a literal string. 

From a test I did with a personal (non-shopify) package:
![Screenshot 2023-08-22 at 11 30 46 AM](https://github.com/Shopify/ui-extensions/assets/12719665/a3074a01-ecd5-4a4b-ba4c-40db482ebee8).

All good though, I can update this to do a conditional step using `npm dist-tag` to apply the latest tag.

Reply:
If you use `npm dist-tag` you need the actual package version and have to run it for the react package too. Is that easy to get?